google/cloud/spanner/value.h

```diff
@@ -203,12 +203,20 @@ class Value {
     Bytes& operator=(Bytes&&) = default;
     ///@}
 
-    /// @name Equality
+    /// @name Relational operators
     ///@{
     friend bool operator==(Bytes const& a, Bytes const& b) {
       return a.data == b.data;
     }
     friend bool operator!=(Bytes const& a, Bytes const& b) { return !(a == b); }
+    friend bool operator<(Bytes const& a, Bytes const& b) {
+      // Note that std::string::operator<() behaves as if the chars were
+      // unsigned, which is exactly how Spanner BYTES values compare.
```

Comment:
This comment seems like it should be in the `Bytes` class docs for users to see.

Reply:
Moved up.